### ISSUE TITLE
Task manager

### ISSUE THREAD

**Issue by mecligge** [Informationretrieval2016/PinterestCrawler#2]
- [ ] Model evaluation - Bo wang

- [ ] Modify UML diagram

- [ ] Improve Business model

- [ ] Improve problem description (divide problem into subproblems like Alessandro Bozzon suggested)

- [ ] Add query description (like make concrete what  the user can search for, as suggested by Alessandro Bozzon )
- [ ] Explain why and how we are going to use Pinterest 

- [ ] Explain more in detail how our ranking and relevance is going to be done.

**Comment by bwanglzu** [comment]
got it.

**Comment by Quispie** [comment]
-[ ]Crawler
-[ ]Data Preprocessing/Index
-[ ]Ranking
-[ ]FrontEnd Web Page

**Comment by sambit2** [comment]
@bwanglzu I think Boolean model will be easier for ranking

**Comment by mecligge** [comment]
@bwanglzu Did you incorporate **User Engagement** as a metric?

**Comment by bwanglzu** [comment]
@mecligge yes, this part is in yesterday's slides, i don't know whether it's appropriate, but to some extend, system can be evaluate by user engagement.

**Comment by bwanglzu** [comment]
@sambit2 I'm reading the ranking part, will give u answer this afternoon.

**Comment by mecligge** [comment]
Nice

**Comment by Quispie** [comment]
I just made a couple of changes to the UML diagrams in the Drive file. I'm not really convinced that the data model diagram is correct, maybe you could have a look and give me some feedback.

**Comment by bwanglzu** [comment]
@Quispie I'll review it after dinner. @sambit2 I've modified relevance and ranking, croudsourcing part for a bit, please check it.

**Comment by sambit2** [comment]
@bwanglzu ok checking

**Comment by mecligge** [comment]
@Quispie  maybe add crowdsourcing element?

**Comment by Temila** [comment]
I think maybe we should mention that clicking on related images reinforces the ranking between two images with more detail In the relevance part @sambit2

**Comment by sambit2** [comment]
@Temila Seeing to it
